test/configparser.cpp

```diff
@@ -44,3 +45,49 @@ TEST_CASE("evaluate_backticks replaces command in backticks with its output",
 			"a single literal backtick: `");
 	}
 }
+
+TEST_CASE("\"unbind-key -a\" removes all key bindings", "[ConfigParser]")
+{
+	ConfigParser cfgparser;
+
+	SECTION("In all contexts by default")
+	{
+		KeyMap keys(KM_NEWSBOAT);
+		cfgparser.register_handler("unbind-key", &keys);
+		cfgparser.parse("data/config-unbind-all");
+
+		for (int i = OP_QUIT; i < OP_NB_MAX; ++i) {
+			if (i == OP_OPENALLUNREADINBROWSER ||
```

Comment:
I think this condition can be removed in that circumstance. These operations weren't bound to a key previously and after unbinding everything now they won't be either.

Reply:
Yep, copy-paste is bad for ya.